unity-renderer/Assets/UIComponents/Tests/ChatNotificationMessageComponentViewTest.cs

```diff
@@ -0,0 +1,174 @@
+using NUnit.Framework;
+using UnityEngine;
+
+public class ChatNotificationMessageComponentViewTest
+{
+    private ChatNotificationMessageComponentView chatNotificationComponent;
+    private Texture2D testTexture;
+    private Sprite testSprite;
+
+    [SetUp]
+    public void SetUp()
+    {
+        chatNotificationComponent = BaseComponentView.Create<ChatNotificationMessageComponentView>("ChatNotificationMessage");
+        testTexture = new Texture2D(20, 20);
+        testSprite = Sprite.Create(testTexture, new Rect(), Vector2.zero);
+    }
+
+    [TearDown]
+    public void TearDown()
+    {
+        chatNotificationComponent.Dispose();
+        GameObject.Destroy(chatNotificationComponent.gameObject);
```

Comment:
it is better to call a `Dispose` method on the component

Reply:
Wait, I don't understand, I'm calling the dispose on the component

Reply:
the `BaseComponentView` already destroys the gameObject, this line should not be necessary:
```
public virtual void Dispose()
    {
        DataStore.i.screen.size.OnChange -= OnScreenSizeModified;
        if (!isDestroyed)
            Destroy(gameObject);
    }
```

Reply:
aaaaaah okay, removing it then!